src/integrations/front-end-integration.php

```diff
@@ -53,6 +53,7 @@ public static function get_conditionals() {
 		'Debug\Marker_Open',
 		'Title',
 		'Meta_Description',
+		'Robots',
```

Comment:
Why are you moving this? We shouldn't output this on error pages.

Reply:
(You) decided that for now, we are keeping it the same as before the indexables.